libmscore/instrument.h

```diff
@@ -21,6 +21,7 @@
 #include "clef.h"
 #include <QtGlobal>
 #include <QString>
+#include <QColor>
```

Comment:
Isn't that in all.h already?

Reply:
I didn't know that. QtCreator was acting like there was an error so I included it. But it works without it so I removed it.